components/composer/src/init.spec.js

```diff
@@ -1021,3 +1023,178 @@ describe("Save composer message as draft", () => {
     );
   });
 });
+
+describe("Composer `value` prop", () => {
+  beforeEach(() => {
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/manifest",
+      { fixture: "composer/manifest.json" },
+    ).as("getMiddlewareManifest");
+
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/account", {
+      fixture: "composer/account.json",
+    }).as("getMiddlewareAccount");
+
+    cy.intercept("GET", "/users", [
+      { name: "Test User", email: "tester@test.com" },
+      { name: "Secound Test User", email: "tester2@test.com" },
+    ]).as("getUsers");
+
+    cy.visit("/components/composer/src/cypress.html");
+
+    cy.get("nylas-composer").should("exist").as("composer");
+    cy.get("nylas-composer").shadow().get(".nylas-composer").should("exist");
+  });
+
+  it("Set value.body", () => {
+    cy.wait(["@getMiddlewareManifest", "@getMiddlewareAccount"]);
+    cy.get("@composer")
+      .then((el) => {
+        const composer = el[0];
+        composer.value = {
+          ...composer.value,
+          body: "<b>HTML Body Test</b>",
+        };
+      })
+      .wait(500);
```

Comment:
Same comment about the `wait` here. Would the following work?
```suggestion
      .then(() => {
         cy.get(".html-editor[contenteditable=true]")
            .invoke("prop", "innerHTML")
            .then((html) => {
                 expect(html).to.equal(`<b>HTML Body Test</b>`);
            });
      });
```